### DISCUSSION TITLE
Application cannot be reviewed

### DISCUSSION THREAD

**Discussion by Cinnazeyy** [community/community#56011] [GitHub Education]
### Hi everyone,

Hi! I am a high school student who has been part of the Github Education programme before without any issues. Today I tried reapplying with the same (still valid) student ID and I got the following error:

> Your application cannot be reviewed until you fix the following:
> -  You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

I already tried changing my name to the EXACT same as it is on the student ID, as well as update my bio with the EXACT same school name. I tried updating various other things as well each at a time and reapplied a total of 8 times already, each ended with the same rejection. 
I am at a total loss as for what to do at this point to verify and any help is greatly apprechiated! 
Thanks!

**Comment by FSWei** [comment]
Me too. The exact same situation

**Comment by samus-aran** [comment]
Hi @Cinnazeyy 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect that this may be required in your case. Good luck and let us know how you get on 👍
